src/js/api/organizations-api.js

```diff
@@ -0,0 +1,28 @@
+import Cookies from 'universal-cookie';
+import { unauthorizedRedirect } from '../auth';
+
+const cookies = new Cookies();
+var request = require('superagent');
+
+const Api = {
+  postCancelRequest: (url, userData) => {
```

Comment:
the rest of the api calls mostly stick to general call structures in their naming, since the entire target url is handed over - it would be fitting to do this here as well or just use the `general-api.js` to get this in.

For context: Since most of the api interaction code is the same, I intend to strip down the different files to reuse most of the code whenever possible...

Reply:
I force pushed and moved the API call to `general-api.js` introducing the `post` method, let me know if I understood correctly what you asked me. :)

Reply:
Thank you very much, that was spot on 😃!